router/router.go

```diff
@@ -105,6 +106,20 @@ func Start() error {
 
 	// Create a single webhook
 	http.HandleFunc("/api/admin/webhooks/create", middleware.RequireAdminAuth(admin.CreateWebhook))
+	// Get all access tokens
+	http.HandleFunc("/api/admin/accesstokens", middleware.RequireAdminAuth(admin.GetAccessTokens))
+
+	// Delete a single access token
+	http.HandleFunc("/api/admin/deleteaccesstoken", middleware.RequireAdminAuth(admin.DeleteAccessToken))
+
+	// Create a single access token
+	http.HandleFunc("/api/admin/createaccesstoken", middleware.RequireAdminAuth(admin.CreateAccessToken))
+
+	// Send a system message to chat
+	http.HandleFunc("/api/admin/sendsystemmessage", middleware.RequireAccessToken(models.ScopeCanSendSystemMessages, admin.SendSystemMessage))
```

Comment:
Would it perhaps make sense to differentiate user/system message through a request param instead of separate controllers? (Although that would require moving the scope checking logic so maybe not)